aws-logs-loggroup/pom.xml

```
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>cloudwatchlogs</artifactId>
-            <version>2.13.18</version>
+            <version>2.15.26</version>
```

There have been issues with resource providers using newer AWS SDK versions than the Java plugin is using:
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/267#issuecomment-632796555
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/324

Is the current AWS SDK version the Java plugin is using sufficient?
```suggestion
            <version>2.15.19</version>
```

---

sure, changed it and moved the version to the properties section